hw/ip/clkmgr/data/clkmgr_testplan.hjson

```diff
@@ -249,6 +249,23 @@
       stage: V2
       tests: ["clkmgr_frequency"]
     }
+    {
+      name: regwen
+      desc: '''This tests the behavior of the regwen CSRs.
+
+            When a regwen is clear, any write to CSRs it locks are ignored.
+            Once a regwen is cleared, it will only be set again after a
```

Comment:
Nit: I think this sentence is missing its end! When will it be set? :-)

Reply:
Thanks, done.